content/publishing/email-and-slack-notifications.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Receive artifacts of successful builds and build status updates with email or Slack.
+title: Email and Slack notifications
+weight: 8
+---
+
+## Email
+
+Email publishing settings can be found in **App settings > Publish > Email**.
+
+Email publishing is the only publishing option which is set enabled (`Enable email publishing`) by default. Default email from the service used to log in (Github, Bitbucket, Gitlab) is used. You can fill in multiple emails.
+
+If a build has finished successfully, release notes (if passed) and generated artifacts are published to the provided email.
+
+If a buid failed, failure reason will be sent with the link to check build logs. Check `Publish artifacts even if tests fail` to publish artifacts even when one or more tests fail. Leaving this option unchecked will attach generated atrifacts (if any) only when build was successful will publish only successful.
+
+### MS Teams
+
+To be able to receive an email from Codemagic to your MS Teams account, please make sure to set `Anyone can send emails to this address` in `Get email address` -> `advanced settings`.
```

Comment:
Make sure to mention that this setting is changed in MS teams not codemagic

Reply:
fixed in d11b88d1f35a70c0e25786f0d8401944b9476f2f